docs/tutorials/my-first-custom-action.rst

```diff
@@ -0,0 +1,256 @@
+Your first custom action!
+######################################################
+
+Learn how to automate everything with Robusta, in *15 minutes* or less :)
+
+Goals
+---------------------------------------
+We’re gonna learn how to create a custom playbook using a short and made-up (but realistic) scenario.
```

Comment:
"going to" not "gonna".

"gonna" is slang

Reply:
I would also cut out the unecessary words. E.g:

_We’re going to a custom playbook_ and not
_We’re going learn how to create a custom playbook_